src/main/java/com/teamc/fms/controller/FacilityController.java

```diff
@@ -0,0 +1,44 @@
+package com.teamc.fms.controller;
+
+import com.teamc.fms.entity.Facility;
+import com.teamc.fms.service.FacilityService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/facilities")
+public class FacilityController {
+
+    @Autowired
+    private FacilityService facilityService;
+
+    @GetMapping("/")
+    public List<Facility> getFacilities(){
+        return facilityService.getFacilities();
+    }
+
+    @PostMapping("/")
+    public String addFacility(@RequestBody Facility facility){
+
+        facilityService.addFacility(facility);
+        return "successfully added facility";
+    }
```

Comment:
Return 201 Created and the newly created facility in the resposne body instead of string (REST conventions)